src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    white,
+    red,
+    blue,
+    green,
+    black,
+    pink,
+    yellow
+}
```

Comment:
enum constants [should be uppercased ](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
Ok) andestand. I fix it